src/ray/core_worker/pubsub/publisher.h

```diff
@@ -70,15 +70,17 @@ class SubscriptionIndex {
 /// Abstraction to each subscriber for the coordinator.
 class Subscriber {
  public:
-  explicit Subscriber() {}
+  explicit Subscriber(const uint64_t publish_batch_size)
+      : publish_batch_size_(publish_batch_size) {}
+
   ~Subscriber() = default;
 
   /// Connect to the subscriber. Currently, it means we cache the long polling request to
   /// memory. Once the bidirectional gRPC streaming is enabled, we should replace it.
   ///
   /// \param long_polling_reply_callback reply callback to the long polling request.
   /// \return True if connection is new. False if there were already connections cached.
-  bool Connect(LongPollConnectCallback long_polling_reply_callback);
+  bool ConnectToSubscriber(LongPollConnectCallback long_polling_reply_callback);
```

Comment:
Naming change should've been addressed last time, but I forgot about it for some reasons.